data/sles4sap/qe_sap_deployment/qesap_azure.yaml

```diff
@@ -5,10 +5,10 @@ terraform:
     az_region: "%REGION%"
     deployment_name: "%DEPLOYMENTNAME%"
     os_image: "%QESAP_CLUSTER_OS_VER%"
-    private_key: "%SSH_KEY_PRIV%"
     public_key: "%SSH_KEY_PUB%"
     hana_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
     iscsi_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
+    hana_instancetype: "%HANA_INSTANCE_TYPE%"
```

Comment:
`qe-sap-deployment/terraform/azure` does not have variable like `hana_instancetype`. It is an AWS only variable.